tests/mountingOptions/mocks.spec.ts

```diff
@@ -37,15 +37,18 @@ describe('mocks', () => {
         </div>
       `,
       computed: {
```

Comment:
Does anyone know the story with Options API? I wonder what the practice is here - maybe `(this.$router as RouterRoute)` (or whatever the interface is)

I don't care too much about the typing on the unit tests either way, mainly the code in `src`... so don't worry about those ones too much, just interested

Reply:
I would say `defineCOmponent` + somthing like : 
```ts
declare module '@vue/runtime-core' {
   interface ComponentCustomProperties {
    $router: Router
  }
}
```

Reply:
`defineComponent()` and `mount()`'s typing is actual magic 🔥